apps/src/templates/manageStudents/ShowSecret.jsx

```diff
@@ -143,22 +149,14 @@ class ShowSecret extends Component {
                 style={styles.image}
               />
             )}
-            <span data-for={tooltipId} data-tip>
```

Comment:
This view is no longer reachable, so the tool tip is being removed. This view was presented when a user would attempt to reset a teacher's picture password. However with the new changes, the button that would show that option is disabled if that user is a teacher.